content/docs/user-guide/build/templates.md

```diff
@@ -0,0 +1,92 @@
+---
+linktitle: Project and Gem Templates
+title: Project and Gem Templates
+description: Learn about the project and gem templates available in O3DE
+weight: 100
+toc: true
+---
+
+O3DE includes a template system, which you can use to create new projects, new gems, or new snippets of code for the engine based on those templates.  The default engine from installer or github includes a few basic templates, but more can be downloaded as part of gems or repositories and added to the engine.
+
+You can find all the existing templates in the root of the engine in the `templates` sub-folder.  They operate by copying their template files into a folder of the user's choosing and replacing certain placeholder text in those files, such as project name, with the actual name given by the user.
+
+## Project templates included with the engine
+These templates are for entire projects, which can be created using the Project Manager and then opened to work on in the Editor.
+For a guide on how to create projects from template, see [Creating Projects Using Project Manager](/docs/welcome-guide/create/creating-projects-using-project-manager). 
+
+### DefaultProject
+This is a good all-round starting point for a game project using O3DE.  It comes with a script to export the project into a shippable standalone package.  It has a number of gems active by default, including ones for UI, basic prototyping, Script Canvas and Animation, among others.  Using this template requires a C++ compiler installed and it includes a basic c++ game module that can be used to write core game logic and components.
+
+### ScriptOnlyProject
+This has an identical gem selection to DefaultProject, but disables all C++ compiling and linking.  The project is still able to use the Lua and Script Canvas capabilities of the engine, but cannot have custom c++ code embedded or use other gems or modules that are not pre-compiled.  It can be used to get a quick look at the engine and editor without having to compile a project, download 3rd party libraries, or install a compiler or linker.   You can convert a Script Only project to a regular project at a later time. 
+
+See the [Script-only 'Quick Start' Projects](/docs/user-guide/build/script-only-projects.md) documentation for more information about this feature.
+
+### MinimalProject
+This has as few gems active as possible, and is a bare-bones starting point.  It is not recommended to use this template as-is, but rather start with it and then activate what gems you might need for your project.   Because it activates as little as possible, and includes as few other gems as possible, it has the quickest startup time and asset compilation time.
+
+## Notable gem templates included with the engine
```

Comment:
```suggestion
## Notable Gem templates included with the engine
```
I believe we usually capitalize the word Gem in headers.

Reply:
Will fix!